ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -330,12 +350,20 @@ open class SearchBar: UIView {
         constraints.append(searchTextField.centerYAnchor.constraint(equalTo: searchTextFieldBackgroundView.centerYAnchor))
         constraints.append(searchTextField.heightAnchor.constraint(equalTo: searchTextFieldBackgroundView.heightAnchor, constant: -2 * Constants.searchTextFieldVerticalInset))
 
+        //progressSpinner
+        searchTextFieldBackgroundView.addSubview(progressSpinner)
+        progressSpinner.translatesAutoresizingMaskIntoConstraints = false
+
+        constraints.append(progressSpinner.leadingAnchor.constraint(equalTo: searchTextField.trailingAnchor, constant: Constants.clearButtonLeadingInset))
+        constraints.append(progressSpinner.heightAnchor.constraint(equalTo: clearButton.widthAnchor))
+        constraints.append(progressSpinner.trailingAnchor.constraint(equalTo: searchTextFieldBackgroundView.trailingAnchor, constant: -1 * Constants.clearButtonTrailingInset))
+        constraints.append(progressSpinner.centerYAnchor.constraint(equalTo: searchTextFieldBackgroundView.centerYAnchor))
+
         //clearButton
         searchTextFieldBackgroundView.addSubview(clearButton)
         clearButton.translatesAutoresizingMaskIntoConstraints = false
 
         constraints.append(clearButton.leadingAnchor.constraint(equalTo: searchTextField.trailingAnchor, constant: Constants.clearButtonLeadingInset))
-        constraints.append(clearButton.widthAnchor.constraint(equalToConstant: Constants.clearButtonWidth))
         constraints.append(clearButton.heightAnchor.constraint(equalTo: clearButton.widthAnchor))
         constraints.append(clearButton.trailingAnchor.constraint(equalTo: searchTextFieldBackgroundView.trailingAnchor, constant: -1 * Constants.clearButtonTrailingInset))
```

Comment:
I think this might need to change since the spinner is touching the edge of the search bar when it's visible.

Reply:
I think setting clearButtonTrailingInset to 10.0 is the ideal change here.

Reply:
Done

---
In reply to: [470774158](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470774158) [](ancestors = 470774158)

Reply:
Fixed

---
In reply to: [470771214](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470771214) [](ancestors = 470771214)